changelogs/unreleased/add-alpine-3_16_ci-cd.md

```diff
@@ -0,0 +1,3 @@
+## feature/build
+
+* Support Alpine 3.16 build.
```

Comment:
Looks like you add only CI, not CI/CD. So the file should be named `changelogs/unreleased/add-alpine-3_16_ci.md`.